taskiq/result.py

```diff
@@ -15,3 +33,37 @@ class TaskiqResult(GenericModel, Generic[_ReturnType]):
     log: Optional[str] = None
     return_value: _ReturnType
     execution_time: float
+
+    error: Optional[BaseException] = None
+
+    class Config:
+        arbitrary_types_allowed = True
+        json_dumps = _json_dumps  # type: ignore
+        json_loads = json.loads
+
+    def raise_for_error(self) -> "Self":
```

Comment:
I think it's worth making exception throwing configurable (if this option is disabled, it will return an exception object as a result). Throwing system exceptions on the client that occurred in the worker can lead to unpredictable results.

Reply:
But you can skip calling this function on client side. For example:

```python
task = await my_task.kiq()
res = await task.wait_result()
if task.is_err:
	print(f"Oh no, we failed. Exc: {res.error}")
```

If you use it this way, it won't be thrown. But if you want to raise it, you can change the code to this:

```python
task = await my_task.kiq()
res = (await task.wait_result()).raise_error() # Error is raised here.
```